politeiawww/quiesce.go

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2020 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"encoding/hex"
+	"net/http"
+
+	pd "github.com/decred/politeia/politeiad/api/v1"
+	www2 "github.com/decred/politeia/politeiawww/api/www/v2"
+	"github.com/decred/politeia/util"
+)
+
+func (p *politeiawww) toggleQuiesce() (*www2.QuiesceReply, error) {
+	p.bbMtx.Lock()
```

Comment:
This `bbMtx` is specific to the best block cache.  Just use the general politeiawww mutex.  `p.Lock()` and `p.Unlock()`.

Locking code is sensitive so put them in helper functions so you don't have to reproduce the locking code. Something like `p.quiesceOn()` and `p.quiesceOff`.